spec/units/services/content_change_deduplicator_service_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe ContentChangeDeduplicatorService do
+  it "removes duplicates preserving only the latest" do
+    content_changes = [
+      build(
+        :content_change, id: 1,
+        public_updated_at: 10.day.ago,
+        content_id: "8e1fef7b-cd23-4dd2-84d0-61955203db61"
+      ),
+      latest_content_change = build(
+        :content_change,
+        id: 2,
```

Comment:
I reckon we don't need to explicitly set the IDs here (saves a few lines 😄)

Reply:
👍